hack/helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
Out of interest, is there a canonical version of this file? If so, it might be useful to link to it in a comment.

Reply:
this is the current latest that gets generated by `helm create chart`

Reply:
Ah got it. Maybe add a "Generated by 'helm create chart'" comment unless that'll be overwritten in the future?